plugins/python/plang/Environment.cpp

```diff
@@ -66,14 +66,37 @@ EnvironmentPtr Environment::get()
     return &g_environment;
 }
 
+
+#if (PY_VERSION_HEX >= 0x03000000)
+int wake_up_numpy()
+#else
+void wake_up_numpy()
+#endif
+{
+    import_array();
+#if (PY_VERSION_HEX >= 0x03000000)
+    return 0;
+#else
+#endif
+}
+
```

Comment:
I think I cleaned all of the dependency stuff up so that you don't need to have this function.